gfx/UltimateCataclysm/pngs_human_body_32x36/mods/Xedra_Evolved/Overlay/Worn/Aura_force/aura_force.json

```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "id": [ "overlay_male_worn_aura_force", "overlay_female_worn_aura_force" ],
+    "fg": "aura_force_mf"
+  }
```

Comment:
Just name the sprite `overlay_worn_aura_force` and same for `overlay_worn_aura_force_on` and remove the json

Reply:
Uops